java/values.yaml

```diff
@@ -1,9 +1,9 @@
 applicationPort: 4550
 image: hmctssandbox.azurecr.io/hmcts/spring-boot-template
 memoryRequests: '512Mi'
-cpuRequests: '100m'
+cpuRequests: '25m'
 memoryLimits: '1024Mi'
-cpuLimits: '2500m'
+cpuLimits: '1000m'
```

Comment:
I would leave the limit - throttling it will just make the app slower to start up